### ISSUE TITLE
Dependency name in AMD module is inconsistent with NPM package name

### ISSUE THREAD

**Issue by cmrd-senya** [strophe/strophejs-plugin-ping#1]
The only external dependency for this package is (no surprise) Strophe.js. 

NPM package for Strophe.js is named `strophe.js` and it is correctly referenced in the `package.json` file. At the same time, the AMD module dependency [is referenced](https://github.com/strophe/strophejs-plugin-ping/blob/43d20cbe4c07641d98bb6ce085708f292e0786a1/strophe.ping.js#L18) as `strophe`.

The problem with it is that actual applications have to make aliases from `strophe.js` to `strophe` in order to make it find the correct dependency. Converse.js [does it](https://github.com/conversejs/converse.js/blob/e31373a6ae37a1859619bc6dee16bfb666a8db3f/src/config.js#L51) in its RequireJS configuration file.

For Webpack bundles it forces application developer to use `resolve.alias` in order to alias `strophe.js` as `strophe`. (See [my demo](https://github.com/cmrd-senya/converse.js-vue-example/blob/56b55e894b94df6e4c9b28f0a6879945cd6b8f8c/build/webpack.base.conf.js#L30)).

It is quite inconvenient because it requires extra configuration for the plugin user application to get started with something which builds correctly.

At the same time it is not needed anymore if you just replace `strophe` with `strophe.js` on the dependency list:
```diff
diff --git a/strophe.ping.js b/strophe.ping.js
index d1f0b33..3a94315 100644
--- a/strophe.ping.js
+++ b/strophe.ping.js
@@ -15,7 +15,7 @@
     if (typeof define === 'function' && define.amd) {
         // AMD. Register as an anonymous module.
         define([
-            "strophe"
+            "strophe.js"
         ], function (Strophe) {
             factory(
                 Strophe.Strophe,
```

It works no worse with Webpack and I believe other bundlers will not be a problem too.

This problem exists with other Converse.js dependencies too, including `strophejs-plugin-vcard`, `strophejs-plugin-disco` and perhaps some others.

**Comment by BLouzada** [comment]
Same problem, fix please.

**Comment by stoyandimov** [comment]
Same problem (bump)

**Comment by sualko** [comment]
You find the merged pr in the current v0.0.3 release now.
